lib/domain/upvote.rb

```diff
@@ -0,0 +1,23 @@
+module Domain
+  class Upvote
```

Comment:
Actually, this PR solves the problem that I want to learn DDD. I didn't realize I created PR for main repo(I just wanted to show it to someone and get some feedback about code). This PR doesn't make sense actually atm.